src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const allTrs = document.querySelectorAll('table tr');
+
+for (const tr of allTrs) {
+  const thElements = tr.getElementsByTagName('th');
+
+  if (thElements.length !== 0) {
+    const copyiedElement = document.createElement('th');
+
+    copyiedElement.innerText = thElements[1].innerText;
+    tr.append(copyiedElement);
+  } else {
+    const tdElements = tr.getElementsByTagName('td');
+    const copyiedElement = document.createElement('td');
+
+    copyiedElement.innerText = tdElements[1].innerText;
+    tr.append(copyiedElement);
+  }
```

Comment:
try to simplify.
use cloneNode method for tr.children[1]  and insertBefore for setting in exact place.

one test failed
expected
![Screenshot from 2023-05-17 12-49-23](https://github.com/mate-academy/js_task_clone_table_column_DOM/assets/33029864/b383b9ef-f689-417b-ba3b-d85a52c0e9e8)

actual:
![Screenshot from 2023-05-17 12-48-54](https://github.com/mate-academy/js_task_clone_table_column_DOM/assets/33029864/4a39c4c4-89c1-4c07-a5da-a1b385d08f29)